internal/cmd/marketplace/marketplace_item.go

```diff
@@ -0,0 +1,377 @@
+	DefaultHeaders                       []AdditionalContainerDefaultHeader     `json:"defaultHeaders,omitempty"  yaml:"defaultHeaders,omitempty"`
+	DefaultLabels                        []AdditionalContainerDefaultLabel      `json:"defaultLabels,omitempty"  yaml:"defaultLabels,omitempty"`
+	DefaultLogParser                     *string                                `json:"defaultLogParser,omitempty"  yaml:"defaultLogParser,omitempty"`
+	DefaultMonitoring                    *AdditionalContainerDefaultMonitoring  `json:"defaultMonitoring,omitempty"  yaml:"defaultMonitoring,omitempty"`
+	DefaultProbes                        *AdditionalContainerDefaultProbes      `json:"defaultProbes,omitempty"  yaml:"defaultProbes,omitempty"`
+	DefaultResources                     *AdditionalContainerDefaultResources   `json:"defaultResources,omitempty"  yaml:"defaultResources,omitempty"`
+	DefaultSecrets                       []AdditionalContainerDefaultSecret     `json:"defaultSecrets,omitempty"  yaml:"defaultSecrets,omitempty"`
+	DefaultTerminationGracePeriodSeconds *float64                               `json:"defaultTerminationGracePeriodSeconds,omitempty"  yaml:"defaultTerminationGracePeriodSeconds,omitempty"`
+	Type                                 *string                                `json:"type,omitempty"  yaml:"type,omitempty"`
+}
+
+type AdditionalContainerContainerPort struct {
+	From     interface{} `json:"from"  yaml:"from"`
+	Name     string      `json:"name"  yaml:"name"`
+	Protocol *string     `json:"protocol,omitempty"  yaml:"protocol,omitempty"`
+	To       interface{} `json:"to"  yaml:"to"`
+}
+
+type AdditionalContainerCustomFilesConfig struct {
+	FileName *string `json:"fileName,omitempty"  yaml:"fileName,omitempty"`
+	FilePath *string `json:"filePath,omitempty"  yaml:"filePath,omitempty"`
+	FileType *string `json:"fileType,omitempty"  yaml:"fileType,omitempty"`
+	Ref      *string `json:"ref,omitempty"  yaml:"ref,omitempty"`
+}
+
+type AdditionalContainerDefaultAnnotation struct {
+	Description *string `json:"description,omitempty"  yaml:"description,omitempty"`
+	Name        string  `json:"name"  yaml:"name"`
+	ReadOnly    *bool   `json:"readOnly,omitempty"  yaml:"readOnly,omitempty"`
+	Value       string  `json:"value"  yaml:"value"`
+}
+
+type AdditionalContainerDefaultConfigMap struct {
+	Files          []PurpleFile `json:"files,omitempty"  yaml:"files,omitempty"`
+	Link           *PurpleLink  `json:"link,omitempty"  yaml:"link,omitempty"`
+	MountPath      *string      `json:"mountPath,omitempty"  yaml:"mountPath,omitempty"`
+	Name           *string      `json:"name,omitempty"  yaml:"name,omitempty"`
+	SubPaths       []string     `json:"subPaths,omitempty"  yaml:"subPaths,omitempty"`
+	ViewAsReadOnly *bool        `json:"viewAsReadOnly,omitempty"  yaml:"viewAsReadOnly,omitempty"`
+}
+
+type PurpleFile struct {
```

Comment:
purple?

Reply:
my bad, 2 types with the same name while writing the schema